docs/documentation/telemetry.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: documentation
+title: Telemetry
+doc_order: 100
+description: About the use of Telemetry in mozregression
+---
+
+# Telemetry
+
+Unless explicitly disabled, mozregression will send an anonymized telemetry "ping" (a small JSON-document) to Mozilla's servers on each use (command-line invocation in the case of console mozregression, each bisection or single run in the case of the GUI).
```

Comment:
The anonymization part is still around :-P See [this comment](https://github.com/mozilla/mozregression/pull/569#discussion_r399119612).